package/src/components/AddressForm/v1/AddressForm.js

```diff
@@ -0,0 +1,304 @@
+         */
+      value: PropTypes.string
+    })),
+    /**
+     * Can the form entry be canceled, when form is being used to edit an address
+     */
+    isCancellable: PropTypes.bool,
+    /**
+     * Cancel event callback
+     */
+    onCancel: PropTypes.func,
+    /**
+     * Country change event callback, used to fetch new list of regions if country has changed.
+     */
+    onCountryChange: PropTypes.func,
+    /**
+     * Form submit event callback
+     */
+    onSubmit: PropTypes.func,
+    /**
+     * Region options
+     */
+    regions: PropTypes.arrayOf(PropTypes.shape({
+      /**
+         * Region option label ("Louisiana", "California")
+         */
+      label: PropTypes.string,
+      /**
+         * Region option value ("LA", "CA")
+         */
+      value: PropTypes.string
+    })),
+    /**
+     * Save/Submit button text.
+     */
+    saveButtonText: PropTypes.string
+  };
+
+  static defaultProps = {
+    address: {
+      address1: "",
+      address2: "",
+      country: "",
+      city: "",
+      firstName: "",
+      lastName: "",
+      postal: "",
+      region: "",
+      phone: ""
+    },
+    components: {
+      ButtonComponent: "Button",
+      PhoneInputComponent: "Phone Input",
+      SelectInputComponent: "Select Input",
+      TextInputComponent: "Text Input"
+    },
+    onCancel() {},
+    onCountryChange() {},
+    onSubmit() {},
+    saveButtonText: "Save and continue"
+  };
+
+  state = {};
+
+  _form = null;
+
+  handleCountryChange = (country) => {
+    const { onCountryChange } = this.props;
+    onCountryChange(country);
+  };
+
+  handleCancel = () => {
+    const { onCancel } = this.props;
+    onCancel();
+  };
+
+  handleSubmit = (address) => {
+    const { onSubmit } = this.props;
+    onSubmit(address);
+  };
+
+  render() {
+    const {
+      address,
+      components: { TextInputComponent, SelectInputComponent, PhoneInputComponent, ButtonComponent },
+      countries,
+      isCancellable,
+      regions,
+      saveButtonText
+    } = this.props;
+    return (
+      <Fragment>
+        <Form
+          ref={(formEl) => {
+            this._form = formEl;
+          }}
+          onSubmit={this.handleSubmit}
+          validator={requiredValidator}
+        >
+          <Grid>
+            <ColFull>
+              <Field name="country" label="Country" isRequired>
+                <SelectInputComponent
+                  name="country"
+                  onChange={this.handleCountryChange}
+                  options={countries}
+                  placeholder="Country"
+                  isSearchable
+                  value={address.country}
+                />
+                <ErrorsBlock names={["country"]} />
+              </Field>
+            </ColFull>
+
+            <ColHalf>
+              <Field name="firstName" label="First Name" isRequired>
+                <TextInputComponent name="firstName" placeholder="First Name" value={address.firstName} />
```

Comment:
@nnnnat You should not pass `value` to any of the inputs. You should pass only `value={address}` to the `Form` and then let ReactoForm manage the values. This might resolve the errors after first submit.

Reply:
yep that fixed it